src/pf-3-form-fields/form-fields.js

```diff
@@ -47,12 +47,15 @@ const selectComponent = ({
       className={ `final-form-select ${invalid ? 'has-error' : ''}` }
       styles={ customStyles }
       { ...input }
-      options={options.filter(option => option.hasOwnProperty('value'))} // eslint-disable-line
+      options={options.filter(option => option.hasOwnProperty('value') && option.value !== null)} // eslint-disable-line
       placeholder={ placeholder || __('Please choose') }
-      value={ options.filter(({ value }) => value === input.value) }
-      multi={ false }
-      isSearchable={ false }
-      isClearable={ !isRequired }
+      value={ options.filter(({ value }) => rest.multi ? input.value.includes(value) : value === input.value) }
+      isMulti={ rest.multi }
+      isSearchable={ !!isSearchable }
```

Comment:
!!isSearchable = isSearchable ?

Reply:
because undefined != false, we need `true` or `false` values, undefined/null won't do it.
see: https://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose.